### ISSUE TITLE
Feature request: Clicking a tag in page body should provide a way to list all pages of that tag 

### ISSUE THREAD

**Issue by pfalcon** [zim-desktop-wiki/zim-desktop-wiki#1247]
This seems to be pretty obvious feature: when seeing @tag in the page body and clicking on it (perhaps, with a control key modifier, though that's much less obvious to users), or pressing Enter (perhaps, with a modifier) should somehow list all pages of that tag. The keyword is "somehow" - there shouldn't be drastic change of context, e.g. if I'm in the middle of editing a page and clicked a tag by mistake, I shouldn't be taken far away from my half-edited page in a way that it's hard to come page.

2 obvious way to deal with that:

1. Open list of pages with the tag in the side pane
2. Open a dialog

Arguably, 1st way is preferrable, and Zim already has all the GUI infrastructure for that.

**Comment by sojusnik** [comment]
Isn't the tag cloud or tag index sufficient for that use-case?

I think that clicking on a text in the page body should be reserved only for links, for consistency reasons.

**Comment by pfalcon** [comment]
> Isn't the tag cloud or tag index sufficient for that use-case?

No. Who would, while reading a page, drop everything, do gazillion of mouse clicks and eyeball movements to find that particular tag in a tag cloud? If it's right here, before your eyes?

> I think that clicking on a text in the page body should be reserved only for links, for consistency reasons.

Conversely, I think that for consistency reasons, anything clickable should be made clickable, to help with user experience. Everyone used twitter, everyone knows what you do with hash tags - you click on them. (Heck, everyone even knows that tags should be [hashtags](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/417)).

**Comment by sojusnik** [comment]
> > Isn't the tag cloud or tag index sufficient for that use-case?
> 
> No. Who would, while reading a page, drop everything, do gazillion of mouse clicks and eyeball movements to find that particular tag in a tag cloud? If it's right here, before your eyes?

I do :) In my case the tag cloud with the tag index is always to the left of the text. It takes only one click to show all pages with one tag, then you can click on another one to see only pages with two tags and so on.

> > I think that clicking on a text in the page body should be reserved only for links, for consistency reasons.
> 
> Conversely, I think that for consistency reasons, anything clickable should be made clickable, to help with user experience. Everyone used twitter, everyone knows what you do with hash tags - you click on them. (Heck, everyone even knows that tags should be [hashtags](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/417)).

I'm with you that hashtags are more common for tags and would love to see them in Zim instead of `@` too.

The other thing is: What exactly should happen if you click on a clickable hashtag. The only solution that comes to my mind is that the tag should be selected in the tag cloud in the side pane. All other solutions with another dialogue etc. are too distracting, IMHO. But then one could argue, why make the hashtag clickable, if the side pane with the tag cloud and index is already open and it already takes one click to filter for tags?

Or maybe you're using the normal index pane (without the tag cloud) and want to open the side pane with tags only occasionally?
